backend/api/migrations/0002_inspection.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.1 on 2017-03-02 01:53
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('api', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Inspection',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('record_date', models.DateField()),
+                ('inspection_date', models.DateField()),
+                ('inspection_type', models.CharField(max_length=255)),
+                ('action', models.CharField(max_length=255)),
+                ('violation_code', models.CharField(max_length=255)),
+                ('violation_description', models.TextField()),
+                ('critical_flag', models.CharField(max_length=255)),
+                ('score', models.SmallIntegerField(null=True)),
+                ('grade', models.CharField(max_length=255)),
```

Comment:
Things like `violation_code` and `grade` sound like they are limited to a small number of possible values. In that case it's more common to use a `ChoicesField` than a `CharField` but it's up to your discretion. There are no hard and fast rules about this but generally less than a dozen possible values means you should at least consider a `ChiocesField`